test-case/check-pause-resume.sh

```diff
@@ -96,18 +96,21 @@ do
     dlogi "Entering expect script with:
       $cmd $SOF_ALSA_OPTS $cmd_opts -D $dev -r $rate -c $channel -f $fmt -vv -i $file_name -q"
 
+    # NOTE: Purposely we don't handle 'Volume MAX' case. So that the MAX output can fall into
+    #       'anything else' case and error out.
+    # FIXME: share this expect script as a common function
```

Comment:
@fredoh9 I re-submitted these comments in new #1222. I think all the rest has been superseded by merged #1218. OK to abandon this?